tasks/bump.js

```diff
@@ -77,8 +82,7 @@ module.exports = function(grunt) {
 
     var globalVersion; // when bumping multiple files
     var gitVersion;    // when bumping using `git describe`
-    var VERSION_REGEXP = /([\'|\"]?version[\'|\"]?[ ]*:[ ]*[\'|\"]?)([\d||A-a|.|-]*)([\'|\"]?)/i;
-
+    var VERSION_REGEXP = /([\'|\"]?version[\'|\"]?[ ]*:[ ]*[\'|\"]?)(\d+\.\d+\.\d+)(-rc\.?\d+)?([\w-]*)?([\'|\"]?)/i;
```

Comment:
With bump:git creating "versions" like `1.0.0-243-g4c8c` the regex above does not work.  With the changes I made to the regex we can recognize a semantic version vs. a `git version`.